docs/release-notes/index.rst

```diff
@@ -49,12 +49,19 @@ Past releases
 Release policy and schedule
 ---------------------------
 
-Canonical is committed to supporting the <"latest major release" or "last two
-major releases"> of Starcraft. <Optional: "We forward-port changes in older
-releases to the latest release, if they're compatible.">
-
-Starcraft is released when it achieves development milestones in its product
-lifecycle. It doesn't follow a predefined release cadence.
+<Apps: Canonical is committed to releasing Starcraft on a regular monthly
+cadence. Our goal is to have new features and fixes available as soon as
+possible. On the first Monday of the month, we publish a major or minor
+version of Starcraft to the candidate channel, where it undergoes testing and
+feedback. If we determine the candidate is ready for mainstream use by the
+Monday of the week after, we release it as a stable version. During testing
+and feedback of a candidate, we prioritise fixes for critical issues.>
```

Comment:
@tigarmo My intention here is to retread what you wrote for Rockcraft.

Reply:
thanks! It works for Rockcraft (of course), but I didn't mean to "impose" that we use this scheduling scheme for all apps.

But that's a discussion that we can have separately of course